src/Identity/test/Identity.FunctionalTests/MapIdentityApiTests.cs

```diff
@@ -110,10 +110,43 @@ public async Task CanLoginWithBearerToken(string addIdentityMode)
         loginResponse.EnsureSuccessStatusCode();
         Assert.False(loginResponse.Headers.Contains(HeaderNames.SetCookie));
 
+        var loginContent = await loginResponse.Content.ReadFromJsonAsync<JsonElement>();
+        var tokenType = loginContent.GetProperty("tokenType").GetString();
+        var accessToken = loginContent.GetProperty("accessToken").GetString();
+        var expiresIn = loginContent.GetProperty("expiresIn").GetDouble();
+        Assert.True(loginContent.TryGetProperty("refreshToken", out _));
+
+        Assert.Equal("Bearer", tokenType);
+        Assert.Equal(3600, expiresIn);
+
+        client.DefaultRequestHeaders.Authorization = new("Bearer", accessToken);
+        Assert.Equal($"Hello, {Email}!", await client.GetStringAsync("/auth/hello"));
+    }
+
+    [Theory]
+    [MemberData(nameof(AddIdentityModes))]
+    public async Task CanChangeAccessTokenResponseJsonOptions(string addIdentityMode)
+    {
+        await using var app = await CreateAppAsync(services =>
+        {
+            AddIdentityActions[addIdentityMode](services);
+            services.ConfigureHttpJsonOptions(options =>
+            {
+                options.SerializerOptions.PropertyNamingPolicy = JsonNamingPolicy.SnakeCaseLower;
+            });
+        });
+        using var client = app.GetTestClient();
+
+        await RegisterAsync(client);
+        var loginResponse = await client.PostAsJsonAsync("/identity/login", new { Email, Password });
+
+        loginResponse.EnsureSuccessStatusCode();
+
         var loginContent = await loginResponse.Content.ReadFromJsonAsync<JsonElement>();
         var tokenType = loginContent.GetProperty("token_type").GetString();
```

Comment:
Didn't these change?

Reply:
Yes, but this test is demonstrating that you can get the old `snake_case` response back by configuring `PropertyNamingPolicy = JsonNamingPolicy.SnakeCaseLower`.